pset-iterators.js

```diff
@@ -114,6 +129,14 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 4:')
 
 // Add your code below this line
+const presidentsBetween1850and1900 = presidents.filter(president => {
+  const startDateInt = president.took_office.split('-')
+  //console.log(startDateInt)
+  if(startDateInt[0] >= 1850 && startDateInt[0] <= 1900) {
```

Comment:
@anniecdavis27 nice algorithm here,  your `if` statement could have also been rewritten as the following: 

```
return startDateInt[0] >= 1850 && startDateInt[0] <= 1900
```

This accomplishes the same result as your logic ... since the callbacks used for `.filter()` needs evaluate to true or false